korio/src/nativeTest/kotlin/com/soywiz/korio/file/std/LocalVfsNativeTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.soywiz.korio.file.std
+
+import com.soywiz.korio.file.*
+import com.soywiz.korio.lang.*
+import kotlin.test.*
+
+class LocalVfsNativeTest {
+
+    @Test
+    fun testUserHomeVfsValue() {
+        val homePath = Environment["HOME"]
+        val expectedResult = if (homePath != null) homePath else "/tmp"
+        assertEquals(
+            expectedResult.replace("/", ""),
+            userHomeVfs.absolutePath.replace("/", "")
```

Comment:
Because expected result had path if form of `/user/home/` and userHomeVfs.absolutePath was in form of `/usr/home`. I'm on macOS and don't know if there are inconsistencies on other OSs.

Reply:
Oh, so maybe better to do `.trimEnd('/')` otherwise you would get userhome that could be /us/er/home or user/home too.

Maybe worth trying to fix that inconsistency too, can you put a reprooducer in an issue to look later?

Reply:
But in any case here I would go for changing it to `.trimEnd('/')` for this PR

Reply:
I agree, `.trimEnd('/')` is better

Reply:
Cool, thanks! Can you update the PR with that change? I will merge it then